### DISCUSSION TITLE
Coupon can't be used with the business plan.

### DISCUSSION THREAD

**Discussion by umairkhanrind** [community/community#123007] [Enterprise] [answered]
Hi, My Current package Github team, Recently i have received the Github enterprise coupon by Microsoft Founders-hub, but when am trying to redeem my coupon its says "Coupon can't be used with the business plan."
Can anyone help me in this matter.

Thanks in advance

**Comment by haakonflender1** [comment]
I have the exact same issue.

**Comment by Tom271** [comment]
I get the same issue with GitHub Free ("Coupon can't be used with the free plan.")

**Comment by angrypiggie** [comment]
I am getting the exact same issue.

**Comment by JerryQian7** [comment]
Having the same issue! I don't see it as activated in my billing plan and now I can't redeem the coupon anymore because it says its been redeemed already.

**Comment by CotherDigi** [comment]
Same problem. I've tried upgrading, personal and organisation profiles, nothing is working. Anyone found a solution yet?

**Comment by umairkhanrind** [comment, marked as answer]
My issue solved through support team. I provided coupon to support team and they applied it on organization.


Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by trparchman-fit** [comment]
I downgraded my organization's Team plan to Free anticipating this would enable me to redeem the benefit.  It seemed to go through, but now my organization's plan is still showing as Free and the redemption site says it can't be used again.

I've got a ticket in with support and fingers crossed that they can get us fixed up quickly.  There are a few features we use that I think will be busted as long as we are on the Free plan.

**Comment by Tom271** [comment]
This was sorted for my organisation through contacting GitHub Support. Now working as expected, we got a response within a day or two.

I recommend anyone hitting this error does the same.

**Comment by mecodeatlas** [comment]
Hey @umairkhanrind 👋 

We’re going through some older unanswered discussions and I know it’s been a while, but I wanted to check in and see if you were able to get this sorted out.
